tests/api-diff/value_tests.t

```diff
@@ -69,3 +69,27 @@ Run api-diff and check the output
   +val f : t -> t
   
   [1]
+
+### Value referencing an abstract type
+
+api-diff should be able to tell two, alias types from both versions of the
```

Comment:
Don't you mean the opposite? We need to make sure non alias types are treated as equal.

Alias types are just resolved so here we're essentially comparing `int` to `int`. I think a more meaningful test would be something like
```
type t = {a: int}
val x : t
```
VS
```
type t = {a: float}
val x : t
```
Wouldn't it? There we would expect a change reported for the type but not for the value and for this to work we have to tell the compiler somehow that `t1` (from ref) is the same as `t2` (from cur) which is done by substituting `t2` by `t1` using `Subst`.

Isn't that what we mean to test here? Alias types are not substituted as far as I can tell.